vllm/v1/outputs.py

```diff
@@ -29,7 +31,7 @@ class ModelRunnerOutput:
     req_id_to_index: Dict[str, int]
 
     # [num_reqs]
-    sampled_token_ids_cpu: torch.Tensor
+    sampled_token_ids_cpu: List[int]
```

Comment:
tiny nit -> this does not need the training `_cpu` suffix. The convention in V1 is that `_cpu` is used to torch cpu objects